mxcubeweb/core/components/samplechanger.py

```diff
@@ -213,10 +213,8 @@ def mount_sample_clean_up(self, sample):
                 elif sc.get_loaded_sample().get_address() != sample["location"]:
                     res = sc.load(sample["sampleID"], wait=True)
 
-                if res is None:
-                    res = True
                 if (
-                    res is not False
+                    not res
```

Comment:
```suggestion
                    res is not None
```
Otherwise the semantics for this if statement changes.

For example `not res` will be True even for res = "", res = 0, not only res = None. Don't know if this actually make a difference in this code.

Reply:
`load` of the `SampleChanger` hardware object should now always return `True` or `False`, it was the case in the past that `None` was returned for a successful load. Your right the test should be changed, but to `if res`. 

Thanks for spotting this one, I made a quick touch up before making the PR and made this mistake but it should be fine now.